content/en/blog/2024/sig-node-spotlight/sig-node-spotlight.md

```diff
@@ -0,0 +1,223 @@
+Node in some way.
+
+**Arpit**: How does SIG Node contribute to Kubernetes' performance and stability?
+
+**M/G/S:** Kubernetes works on nodes of many different sizes and shapes, from small physical VMs
+with cheap hardware to large AI/ML-optimized GPU-enabled nodes. Nodes may stay online for months or
+maybe be short-lived and be preempted at any moment as they are running on excess compute of a cloud
+provider.
+
+[Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) -- the
+Kubernetes agent on a node -- must work in all these environments reliably. As for the performance
+of kubelet operations, this is becoming increasingly important today. On one hand, as Kubernetes is
+being used on extra small nodes more and more often in telecom and retail environments, it needs to
+scale into the smallest footprint possible. On the other hand, with AI/ML workloads where every node
+is extremely expensive, every second of delayed operations can visibly change the price of
+computation.
+
+
+## Challenges and Opportunities
+
+**Arpit:** What upcoming challenges and opportunities is SIG Node keeping an eye on?
+
+**M/G/S:** As Kubernetes enters the second decade of its life, we see a huge demand to support new
+workload types. And SIG Node will play a big role in this. The Sidecar KEP, which we will be talking
+about later, is one of the examples of increased emphasis on supporting new workload types.
+
+The key challenge we will have in the next few years is how to keep innovations while maintaining
+high quality and backward compatibility of existing scenarios. SIG Node will continue to play a
+central role in Kubernetes.
+
+**Arpit:** And are there any ongoing research or development areas within SIG Node that excite you?
+
+**M/G/S:** Supporting new workload types is a fascinating area for us. Our recent exploration of
+sidecar containers is a testament to this. Sidecars offer a versatile solution for enhancing
+application functionality without altering the core codebase.
+
+**Arpit:** What are some of the challenges you've faced while maintaining SIG Node, and how have you
+overcome them?
+
+**M/G/S:** The biggest challenge of SIG Node is its size and the many feature requests it
+receives. We are encouraging more people to join as reviewers and are always open to improving
+processes and addressing feedback. For every release, we run the feedback session at the SIG Node
+meeting and identify problematic areas and action items.
+
+**Arpit:** Are there specific technologies or advancements that SIG Node is closely monitoring or
+integrating into Kubernetes?
+
+**M/G/S:** Developments in components that the SIG depends on, like [container
+runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes/)
+(e.g. [containerd](https://containerd.io/) and [CRI-O](https://cri-o.io/), and OS features are
+something we contribute to and monitor closely. For example, there is an upcoming `cgroup v2`
```

Comment:
Not sure what was said; if the interviewees stand by “cgroup v2” (which is wrong), let's add an editor's note to explain that actually it's the v1 support that will (ie, is likely to) be deprecated.

```suggestion
something we contribute to and monitor closely. For example, there is an upcoming _cgroup v1_
```